docs/modules/vertiq_2306_2200.rst

```diff
@@ -1,15 +1,15 @@
 *********************************
-Vertiq 2306 2200KV | Speed Module
+Vertiq 23-06 | Speed Module
 *********************************
 
-.. sidebar:: Vertiq 2306 2200KV
+.. sidebar:: Vertiq 23-06
 
     .. image:: ../_static/IQ2306_2200kv.png
-        :alt: Vertiq 2306 2200KV
+        :alt: Vertiq 23-06 2200KV
```

Comment:
The 3 lines here where you changed the hyphens, should they lose the 2200 as well?